frontend/src/App.js

```diff
@@ -1,18 +1,27 @@
 import { Route, Routes } from 'react-router-dom';
+import { ToastContainer } from 'react-toastify';
+import Landing from './pages/Landing/Landing';
 import Login from './pages/Login';
 import Signup from './pages/Signup';
-import Landing from './pages/Landing/Landing';
+import ManageUserProfiles from './pages/ManageUserProfiles/ManageUserProfiles';
+import UserProfile from './pages/UserProfile/UserProfile';
 import './App.css';
 import './css/Auth.css';
 
 function App() {
   return (
-    <Routes>
-      <Route path="/login" element={<Login />} />
-      <Route path="/signup" element={<Signup />} />
-      <Route path="/landing" element={<Landing />} />
-      <Route path="/" element={<Login />} /> {/* Display Login component by default */}
-    </Routes>
+    <div>
+      <Routes>
+        <Route path='/login' element={<Login />} />
+        <Route path='/signup' element={<Signup />} />
+        {/* TODO: Create ProtectedRoute to handle the routes for authenticated users */}
+        <Route path='/landing/*' element={<Landing />} /> {/* TODO: create an unauthorised action page */}
```

Comment:
Would `/dashboard/` be a better name to call it?

Reply:
Do you mean renaming `/landing/` to `/dashboard/`?

Reply:
Yup. Is the landing page playing the role of a dashboard? Ie the page the user sees after logging in

Reply:
Yup its the page the user sees after logging in but its a list of questions from assignment 1. I dont think that's a dashboard

Reply:
Yes our assignment 1 requirements identifies that page as Landing Page so I guess we should just stick with that name.